docs/development/workflow/development_workflow.rst

```diff
@@ -194,6 +194,10 @@ still run the checks manually (see, :ref:`code-style`) using::
 
   tox -e codestyle
 
+Or::
+
+  pre-commit run
```

Comment:
@eerovaher , something like this?

Reply:
I think it is worth mentioning that `pre-commit run` does not require `pre-commit` to be installed in the repository.

Reply:
OK, I updated the text. What about now? Thanks!